docs/language/overview.md

```diff
@@ -965,13 +959,17 @@ produces
 -2
 ```
 
+Note that if the expression has side effects,
+as with [aggregate function calls](#610-aggregate-function-calls), only the selected expression
+will be evaluated.
```

Comment:
I moved this "note" after the example because, as originally written, I'd initially expected the example to show something related to what was just described in the note.

Reply:
@nwt: I agree it's important. Therefore, I've added a second example. Please give it another look.